### DISCUSSION TITLE
Updated Debian 12: 12.6 released

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#3300] [Announcements]
https://www.debian.org/News/2024/20240629

**Comment by tteck** [comment]
If you have manually updated to Debian 12.6, execute `rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED` in the LXC console.
The Proxmox VE LXC Updater and Proxmox VE Cron LXC Updater have been modified (6/29) for Debian 12.6.

**Comment by Toolmaster30** [comment]
Thank you very much. Works perfectly.

**Comment by heikone** [comment]
Could anybody briefly describe, how to update from Debian 12.2 to 12.6?
Didn't find any how to so far.
Greetings, Heiko

**Reply by tteck** [reply]
https://tteck.github.io/Proxmox/#proxmox-ve-lxc-updater

**Reply by heikone** [reply]
Great! Thank you!

**Comment by Stoony1** [comment]
THX

**Comment by enreda-net** [comment]
What is the reason why you don't update with a simple apt upgrade?

The container doesn't work properly?

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/3300#discussioncomment-9923938
